lib/js/components/Icon/Icon.consts.ts

```diff
@@ -18,7 +18,7 @@ const BETHINK_ICONS = {
 	RIBBON: Ribbon,
 } as const;
 
-export const ICONS = {
+export let ICONS = {
```

Comment:
why this change?

Reply:
I was sure it was suggestion by IDE, but now I haven't it. Will revert it.